x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.test.tsx

```diff
@@ -230,22 +271,82 @@ describe('DetectionEnginePageComponent', () => {
     });
   });
 
-  it('the pageFiltersUpdateHandler updates status when a multi status filter is passed', async () => {
-    (FilterGroup as jest.Mock).mockImplementationOnce(({ onFilterChange }: FilterGroupProps) => {
-      if (onFilterChange) {
-        // once with status
-        onFilterChange([
-          {
-            meta: {
-              index: 'security-solution-default',
-              key: 'kibana.alert.workflow_status',
-              params: ['open', 'acknowledged'],
-            },
+  it('should pass building block filter to the alert Page Controls', async () => {
+    const MockedFilterGroup = FilterGroup as jest.Mock;
+    MockedFilterGroup.mockImplementationOnce(getMockedFilterGroupWithCustomFilters());
+    await waitFor(() => {
+      render(
+        <TestProviders store={getStoreWithCustomState(stateWithBuildingBlockAlertsEnabled)}>
+          <Router history={mockHistory}>
+            <DetectionEnginePage />
+          </Router>
+        </TestProviders>
+      );
+    });
+
+    expect(MockedFilterGroup.mock.calls[0][0].filters[0]).toMatchObject(
```

Comment:
I wonder if it's possible to use `toHaveBeenCalledWith` here to avoid reaching into the specifics of the mock implementation. 

E.g. 

```js
expect(MockedFilterGroup.toMatchObject(expect.objectContaining({
  filters: expect.objectContaining({
    ...
```

Reply:
Done.